src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -590,221 +561,168 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
                         accountInserts[i].addError(err.getMessage());
                         Contact con = listContactNewAcct[i];
-                        con.addError(string.format(label.conFailedAccountCreate, new string[]{con.Firstname, con.Lastname, err.getMessage()}));
+                        con.addError(String.format(label.conFailedAccountCreate, new String[]{con.Firstname, con.Lastname, err.getMessage()}));
                     }
                 }
             }
-                                    
+
             // now update each contact's accountId
             Integer i = 0;
-            for (Contact c : listContactNewAcct) {
-                c.AccountId = accountInserts[i].Id;
+            for (Contact contactRecord : listContactNewAcct) {
+                contactRecord.AccountId = accountInserts[i].Id;
                 i++;
             }
         }
     }
 
     /*******************************************************************************************************
     * @description Updates an Individual Account for an Individual Contact
-    * @param contacts the list of Contacts that need updating.
-    * @param dmlWrapper to hold the Contacts that need updating.
+    * @param oneToOneContactsToUpdate the list of Contacts that need updating.
     * @return void
-    */ 
-    private void updateContactAccount(Contact[] contacts, DmlWrapper dmlWrapper)
-    {
+    */
+    private void updateOneToOneAccounts(List<Contact> oneToOneContactsToUpdate){
+        if (oneToOneContactsToUpdate.isEmpty()) {
+            return;
+        }
+
         Map<Id,Contact> accConMap = new Map<Id,Contact>();
-        for (Contact c : contacts) {
-            accConMap.put(c.AccountId, c);
+        for (Contact contactRecord : oneToOneContactsToUpdate) {
+            accConMap.put(contactRecord.AccountId, contactRecord);
         }
         //get the corresponding Accounts
         // DJH: note that we explicitly do NOT get the account name.  This is to avoid the changes to the account from
         // this routine which get added to dmlWrapper, don't end up overwriting the name changes that are done syncronously
         // for HH accounts (which don't use dmlWrapper).
-        List<Account> accts = [Select a.npe01__SYSTEM_AccountType__c, a.ShippingStreet, a.ShippingState, a.ShippingPostalCode, a.ShippingCountry, 
+        List<Account> accts = [SELECT a.npe01__SYSTEM_AccountType__c, a.ShippingStreet, a.ShippingState, a.ShippingPostalCode, a.ShippingCountry,
             a.ShippingCity, a.BillingStreet, a.BillingState, a.BillingPostalCode, a.BillingCountry, a.BillingCity
-            From Account a where a.Id IN :accConMap.keyset()];
+            FROM Account a WHERE a.Id IN :accConMap.keyset()];
         List<Account> listAccount = new List<Account>();
-        
-        for (Account a : accts) {         
-            Contact c = accConMap.get(a.Id);
+
+        for (Account a : accts) {
```

Comment:
Wondering if this for loop should iterate over the query directly rather than query into a list and then iterate over the list